ui/src/adapters/parsers/forms.ts

```diff
@@ -319,13 +345,17 @@ export function serializeCredentialLinkIssuance({
   if (serializedSchemaForm.success) {
     return {
       data: {
-        credentialExpiration: credentialExpiration
-          ? serializeDate(credentialExpiration, "date")
-          : null,
+        credentialExpiration: credentialExpiration ? credentialExpiration.toISOString() : null,
```

Comment:
We should keep using `serializeDate` and just pass the right output format, in this case, `"date-time"`.

Reply:
Yes. Change reverted. I also change some other places where to.ISOString() was being used.